gimmicks/Gimmicks.py

```diff
@@ -168,6 +170,10 @@ def export(self, i):
         gimmick_data.append(param5)
 
     def image(self, data):
+        if data.get(self.kind) is not None:
+            if 'path' not in data.get(self.kind):
+                return data.get(self.kind)[self.param0]
```

Comment:
this isn't always true though I don't think...

Reply:
In the edge cases where it isn't there are overrides in place.